CONTRIBUTING.md

```diff
@@ -0,0 +1,166 @@
+# Contributing Guidelines
+
+Thank you for looking for ways to help on Delta Chat iOS!
+
+This document tries to outline some conventions that may not be obvious
+and aims to give a good starting point to new contributors.
+
+
+## Reporting Bugs
+
+If you found a bug, [report it on Github](https://github.com/deltachat/deltachat-ios/issues).
+
+Project maintainers may transfer bugs that are not UI specific
+(eg. network, database or encryption related)
+to [Delta Chat Core](https://github.com/deltachat/deltachat-core-rust/issues).
+If you assume beforehand, that the bug you've found belongs to Core,
+you can report there directly.
+
+Please search both open and closed issues to make sure your bug report is not a duplicate.
+
+For community interactions around Delta Chat
+please read our [Community Standards](https://delta.chat/community-standards).
+
+
+## Proposing Features
+
+If you have a feature request,
+create a new topic on the [Forum](https://support.delta.chat/c/features/6).
+
+
+## Rough UX Philosophy
+
+Some rough ideas, that may be helpful when thinking about how to enhance things:
+
+- Work hard to avoid options and up-front choices.
+  Thinking about concrete user stories may help on that
+- Avoid to speak about keys and other hard to understand things in the primary UI
+- The app shall work offline as well as with bad network
+- Users do not read (much)
+- Consistency matters
+- Offer only things that are highly useful to many people in primary UI.
+  If really needed, bury other things eg. in some menus
+- The app should be for the many, not for the few
```

Comment:
very nice list.  What about adding '.' at the end of statements/sentences? 
makes it more "on point" even if it may be considered "old style" :)

Reply:
point.